ci/build_container/build_recipes/gperftools.sh

```diff
@@ -15,4 +15,6 @@ tar xf gperftools-"$VERSION".tar.gz
 cd gperftools-"$VERSION"
 
 LDFLAGS="-lpthread" ./configure --prefix="$THIRDPARTY_BUILD" --enable-shared=no --enable-frame-pointers --disable-libunwind
```

Comment:
Need `-lc++` here if stdlib is libc++, it overrides existing LDFLAGS and will result failure when I tried to build prebuilt deps with libc++.

Reply:
Huh, it works for me locally and on the CI.

How are you building that, exactly? Can you paste the command and/or environment you're building this in? Thanks!

Reply:
I have slightly modified `build_container_common.sh` to pass CXXFLAGS/LDFLAGS to build the image, and only this one will fail to link (as it compiled with libc++ but not linking).

Reply:
Does changing this to `LDFLAGS="${LDFLAGS} -lpthread"` fix your build?

Reply:
Done, let me know if this works for you.